RockLib.Configuration.All.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 17
-VisualStudioVersion = 17.0.32126.317
+# Visual Studio Version 16
```

Comment:
Did we want to move from 17 to 16?

Reply:
That....wasn't intentional. I'm not sure why that got updated.

Though the other solutions in the configuration repo are at 16 (well, ProxyFactory is at 15 something)